lib/dependencies-parser.ts

```diff
@@ -50,6 +49,8 @@ export interface NugetDependency extends Dependency {
 export interface PaketDependencies extends Array<DependencyGroup> {
 }
 
+const startsWith = (a: string, b: string): boolean => a.indexOf(b) === 0;
```

Comment:
Seems legit, but could we put this into a utils file so we don't unnecessarily repeat code?

Reply:
Uhm, could you use startsWith? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith

Side note, this repo doesn't have an nvmrc